example/src/screens/HomeScreen/index.tsx

```diff
@@ -0,0 +1,175 @@
+    }
+
+    return marginBottom;
+  };
+
+  const onLayoutContent = (title: string) => (e: LayoutChangeEvent) => {
+    const newHeight = e.nativeEvent.layout.height;
+
+    setContentHeight((prevHeight) => {
+      return { ...prevHeight, [title]: newHeight };
+    });
+  };
+
+  return (
+    <>
+      <StatusBar barStyle="light-content" backgroundColor={colors.primaryGreen} translucent />
+      <TabbedHeaderPager
+        containerStyle={screenStyles.stretchContainer}
+        backgroundColor={colors.primaryGreen}
+        tabsContainerBackgroundColor={colors.secondaryGreen}
+        rememberTabScrollPosition
+        logo={require('../../assets/images/logo.png')}
+        title={"Mornin' Mark! \nReady for a quiz?"}
+        titleStyle={screenStyles.text}
+        offscreenPageLimit={5}
+        foregroundImage={require('../../assets/images/photosPortraitMe.png')}
+        tabs={TABS.map((tab) => ({ title: tab.title }))}
+        tabTextStyle={screenStyles.text}
+        // Refresh control is not implemented on web and even if provided, it will double padding top and bottom
+        {...(Platform.OS !== 'web' && {
+          refreshControl: (
+            <RefreshControl
+              //  z Index is required on IOS, to refresh indicator be visible
+              style={styles.refreshControl}
+              refreshing={refreshing}
+              titleColor={colors.white}
+              tintColor={colors.white}
+              title="Refreshing"
+              onRefresh={onRefresh}
+            />
+          ),
+        })}>
+        {TABS.map((tab, i) => {
+          const title = tab.contentTitle;
+          const marginBottom = Platform.select({ ios: calcMargin(title) + 20, android: 10 });
+
+          return (
+            <View
+              key={i}
+              onLayout={onLayoutContent(title)}
+              style={[screenStyles.content, { marginBottom }]}>
+              <Modal
+                animationType="slide"
+                transparent
+                visible={modalVisible}
+                style={styles.modalStyle}>
+                <View style={styles.modalContentContainer}>
+                  <UserModal
+                    setModalVisible={setModalVisible}
+                    onPressCloseModal={() => setModalVisible(false)}
+                    user={userSelected}
+                  />
+                </View>
+              </Modal>
+              <Text style={screenStyles.contentText}>{title}</Text>
+              {users.map(
+                (user) =>
+                  (title === 'Popular Quizes' || title === user.type) && (
+                    <QuizListElement
+                      key={user.author}
+                      elements={user.cardsAmount}
+                      authorName={user.author}
+                      mainText={user.label}
+                      labelText={user.type}
+                      imageSource={user.image}
+                      onPress={() => navigation.navigate('Card', { user })}
```

Comment:
Have you considered moving arrow functions to the handlers outside the return?
I saw an arrow functions like this passed to the components in a few places. It's not necessarily a bad practice, but it could be optimized.

https://reactjs.org/docs/faq-functions.html
"Using an arrow function in render creates a new function each time the component renders, which may break optimizations based on strict identity comparison."

Reply:
Good catch, I was fixing such cases in previous PRs, so probably it was omitted